<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generic_window</class>
 <widget class="QDialog" name="generic_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
border-style: solid;
border-color:  black;
border-width: 1px;
}</string>
  </property>
  <widget class="QLabel" name="prof_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sin usuario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prof_pic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#prof_pic { 
background-image: url(:/iconos/screen800x600/iconos/Prof pic azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_1 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Diagrama.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="num_of_act">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>5</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#num_of_act
{
color: yellow;
border:none;
background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_4 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Campana.png);
border: none; 
background-repeat: none;
background-position: center;

 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_3 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Bitacora azul.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>34</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_2 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Encendido azul.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="top_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>759</width>
     <height>27</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_bar { 
background-color: transparent;
background-image: url(:/barras/screen800x600/barras/Indice diagrama limpio.png);
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_hora">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="label_title">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>-1</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Diagrama</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dia">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>28</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dia</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>530</y>
     <width>70</width>
     <height>67</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton { 
background-color: transparent;
border-image: url(:/iconos/images/Iconos/Menu_azul.png); 
background: none; 
border: none; 
background-repeat: none;
 } 
#pushButton:pressed {
border-image: url(:/iconos/images/Iconos/Menu_azul.png); 
 }
#pushButton:checked{
 border-image: none;
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="active_param_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>580</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phetsarath OT</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#active_param_label { 
background-color: transparent;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string>Par√°metros</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_quimicos">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_quimicos { 
background-image: url(:/iconos/screen800x600/iconos/Matraz azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_electricos">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_electricos { 
background-image: url(:/iconos/screen800x600/iconos/Clavija azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bottom_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>529</y>
     <width>766</width>
     <height>70</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#bottom_bar { 
background-color: transparent;
border-image: url(:/barras/images/Barras/Parametros_azul.png); background: none; 
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_fisicos">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_fisicos { 
background-image: url(:/iconos/screen800x600/iconos/Medidor azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="asa_logo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#asa_logo{ 
background-image: url(:/iconos/screen800x600/iconos/ASA logo.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_main_menu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_main_menu{ 
background-color: transparent;
border: none; 
background-repeat: none;
background-position: center;

 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>bottom_bar</zorder>
  <zorder>top_bar</zorder>
  <zorder>prof_label</zorder>
  <zorder>prof_pic</zorder>
  <zorder>top_menu_1</zorder>
  <zorder>num_of_act</zorder>
  <zorder>top_menu_4</zorder>
  <zorder>top_menu_3</zorder>
  <zorder>top_menu_2</zorder>
  <zorder>label_hora</zorder>
  <zorder>label_title</zorder>
  <zorder>label_dia</zorder>
  <zorder>pushButton</zorder>
  <zorder>active_param_label</zorder>
  <zorder>pb_quimicos</zorder>
  <zorder>pb_electricos</zorder>
  <zorder>pb_fisicos</zorder>
  <zorder>asa_logo</zorder>
  <zorder>top_main_menu</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
